.github/workflows/test.yml

```
@@ -16,11 +16,21 @@ jobs:
         ruby:
           - 2.6
           - 2.7
+          - 3.0
```

Still, `3.0` needs to be quoted, or GHA will test with Ruby 3.2. See https://github.com/zendesk/kasket/actions/runs/4434156832/jobs/7779871418
```suggestion
          - '3.0'
```